docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,174 @@
+
+</div>
+</div>
+</div>
+
+Pure black text on a white background can cause eye strain. Instead, try black text on light blue, pale green, or gray background. The following example demonstrates good color contrasting and has a 14.9.1 contrast ratio.
+
+**Good contrast ratio**
+![Good contrast](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_good_color_contrast.png "Good contrast")
+
+It is essential to avoid colors that are too close together. For example, avoid green and black, green and gray, and green and red. Alternatively, it is good practice to use light text on dark backgrounds or dark text on light backgrounds. The example below shows poor color contrasting because the black text is on a green background. The color constrast ratio is 3.8.1. It would have been better to use white text on a dark green background.
+
+**Poor contrast ratio**
+![Poor contrast](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_poor_color_contrast.png "Poor contrast")
+
+### Text headings
+Text headings organize and provide structure for a page, helping users to understand its contents. Headings should be logical, clear, and concise. The following example demonstrates the incorrect and correct structure of text headings.
+
+![Text headings](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_text_headings.png "Text headings")
+
+Your main heading should be H1 and indicate the main content. The incorrect example displays the main heading as H2, which you should only use for subtopics. In addition, don't skip or alter headings as it could cause a fragmented experience, as shown above. H3 is out of order and should follow H2. The correct structure displayed above on the right shows heading levels from h1 to h3 communicating more specific information about the previous section.
+
+For an example video on how to create text headings, see [How Headings Help Screen Reader Users](https://www.youtube.com/watch?v=zRTqCGJ_HwQ).
+
+### Text size and alignment
+The use of white space around blocks of text makes it easier to read. We recommend the following text formatting:
+* left-aligned text
+* font size of 14px
+* 55-65 characters per line
+* sans serif fonts
+
+The following examples demonstrate the incorrect and correct usage of text size and alignment. The incorrect example shows 25-40 characters per line, uses Comic Sans MS, and the font is 11px. The correct example uses the recommended font size of 14 px, has 55-65 characters per line, and uses Arial font.
```

Comment:
Reword this section to use softer terms than "incorrect" and "correct".
The two examples shown here might need to change. We recommend using a sans-serif font, but Comic Sans, used in the incorrect example, is a sans-serif font. That could be confusing. 

We also don't explain why or how the font size, characters per line, or text alignment contribute to increasing accessibility as extensively as in previous sections.

I would also recommend dividing up the explanations between the incorrect and correct versions of examples so it matches how you structured the examples for color contrast. Adding titles would also help keep the article consistent.